src/drafts/MarkdownEditor/_ViewSwitch.tsx

```diff
@@ -45,7 +45,7 @@ export const ViewSwitch = ({selectedView, onViewSelect, onLoadPreview, disabled}
   return (
     <Box sx={{display: 'flex', flexDirection: 'row'}}>
       {condensed ? (
-        <IconButton {...sharedProps} disabled={disabled} icon={icon} label={label} />
+        <IconButton {...sharedProps} disabled={disabled} icon={icon} aria-label={label} />
```

Comment:
Is `label` something that ever worked for `IconButton`? Was trying to track down this type but `ButtonA11yProps` only seems to include `aria-label` and `aria-labelledby`, if I understand right?

Reply:
I don't think it ever worked either - or at least I didn't see it in a recent changelog